content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+
+
+
+**Properties:**
+
+
+  * `private_key_path`
+    The path where the key should be saved on the machine. If this is a bootstrap process, this refers to the local computer. If this will run on the manager, this will be saved on the manager.
+
+    *type:* string
+    *default:*
+  * `public_key_path`
+    The path to read from existing public key.
+
+    *type:* string
+    *default:*
+  * `user`
+    The user account for this key. A corresponding user account will be created by GCP when the key is added to the Instance. This must be supplied for a non-external resource key. See https://cloud.google.com/compute/docs/instances/adding-removing-ssh-keys
+
+    *type:* string
+    *default:*
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the Google Cloud Platform API.
+
+    *default:* {}
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource.
+
+    *type:* boolean
+    *default:* False
+
+
+
+
+
+## cloudify.gcp.nodes.Network
+**Derived From:** [cloudify.nodes.Network]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP Network. This supports either auto-assigned or manual subnets. Legacy networks are not supported. See the GCP Manager and Networks section below if you plan to run a cloudify manager on GCP.
+
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the Google Cloud Platform API.
+
+    *default:* {}
+  * `auto_subnets`
+    Whether to use the GCP "autoCreateSubnetworks" feature (see https://cloud.google.com/compute/docs/subnetworks#networks_and_subnetworks)
+
+    *default:* True
+  * `additional_settings`
+    Additional setting for network
+
+    *default:* {}
+  * `name`
+    Optional Network name. The instance ID will be used by default.
+
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists or if Cloudify should create the resource.
+
+    *type:* boolean
+    *default:* False
+
+
+
+### Example
+
+{{< gsHighlight  yaml  >}}
+
+my_net:
+  type: cloudify.gcp.nodes.Network
+
+{{< /gsHighlight >}}
+
+
+
+
+
+## cloudify.gcp.nodes.Route
+**Derived From:** [cloudify.nodes.Router]({{< relref "blueprints/built-in-types.md" >}})
+
+A defined route, which will be added to the specified network.
+If tags are specified, it will only be added to Instances matching them.
+
+
+
+**Properties:**
+
+
+  * `dest_range`
+    The outgoing range that this route will handle
+
+    *required* None
+  * `priority`
+    The routing table priority for this route. Routes with lower priority numbers will be chosen first if more than one route with a matching prefix of the same length.
+
+    *default:* 1000
+  * `additional_settings`
+    Additional setting for firewall
+
+    *default:* {}
+  * `next_hop`
+    The Instance, IP or VpnTunnel which will handle the matching packets
```

Comment:
The Instance, IP or VpnTunnel ~~which~~ **that** will handle the matching packets **.**